sumo/cli/bandplot.py

```diff
@@ -306,9 +306,22 @@ def bandplot(filenames=None, code='vasp', prefix=None, directory=None,
             dos, pdos = load_dos(dos_file, elements, lm_orbitals, atoms, gaussian,
                                  total_only)
         elif code == 'castep':
-            dos = read_castep_tdos(dos_file, gaussian=gaussian,
-                                   efermi_to_vbm=True)
-            pdos = {}
+            pdos_file = None
+            if cell_file:
+                pdos_file = cell_file.replace('.cell', '.pdos_bin')
+                if not os.path.isfile(pdos_file):
+                    pdos_file = None
+                    logging.info(f"Selecting PDOS file {pdos_file} does not exis, falling back to TDOS.")
+                else:
+                    logging.info(f"Selecting PDOS file {pdos_file}")
+            else:
+                logging.info(f"Cell file {cell_file} does not exis, cannot plot PDOS.")
+
+            dos, pdos = read_castep_dos(dos_file, pdos_file=pdos_file, cell_file=cell_file, 
+                                  gaussian=gaussian,
+                                  lm_orbitals=lm_orbitals,
+                                  elements=elements,
+                                  efermi_to_vbm=True)
```

Comment:
I think this option needs tweaking to respect user args. Not your fault, was already there!

Reply:
I guess you mean the `efermi_to_vbm`? There seem to be no options about it in the outer function...

Reply:
Yup, I think an overhaul of Fermi level handling in Sumo needs to be follow-up work on this. Let's get this PR done first.